api/pacman-api-asset/src/main/java/com/tmobile/pacman/api/asset/controller/AssetListController.java

```diff
@@ -566,8 +567,27 @@ public ResponseEntity<Object> getExemptFilterValue(@PathVariable String attribut
         if (request == null || Strings.isNullOrEmpty(request.getAg())) {
             return ResponseUtils.buildFailureResponse(new Exception("Asset group is Mandatory"));
         }
-        Map<String,Object> response=new HashMap<>();
-        response.put("response",assetService.getAssetExemptedFilterValue(request, attribute));
+        Map<String, Object> response = new HashMap<>();
+        try {
+            response.put("response", assetService.getAssetExemptedFilterValue(request, attribute));
+        } catch (Exception e) {
+            LOGGER.error("Exception in formResponseWithCount ", e);
+            return ResponseUtils.buildFailureResponse(e);
+        }
+        return ResponseUtils.buildSucessResponse(response);
+    }
+    @RequestMapping(path = "/v1/getCompliantFilterValue", method = RequestMethod.POST)
```

Comment:
Is `AssetController` perfect place for that?
If it is then at least path must reflect it `/v1/assets/getCompliantFilterValue`